analysis/coverage_data_utils.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions for coverage data calculation."""
+
+import math
+import posixpath
+from collections import defaultdict
+import pandas as pd
+
+from common import filestore_utils
+from common import experiment_utils as exp_utils
+
+_DEFAULT_RARE_REGION_THRESHOLD = 0
+
+
+def download_json_summary(experiment_name, dst_file):
```

Comment:
nit: consider calling this copy instead of download, because the filestore could be a local path as well.